maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -191,10 +191,20 @@ else if(isCreatePlatform) {
             }
         }
 
-        wizard.promptForJavaHomeIfMissing(server);
 
         server.setUnspecifiedToDefault();
         configureVersion(server, isCreatePlatform);
+
+        String platformVersion = server.getPlatformVersion();
+        if (platformVersion.charAt(0) == 1) {
+            wizard.showMessage("Note: JDK 1.7 is needed for platform version " + platformVersion + ".");
```

Comment:
@rkorytkowski What about earlier versions of JDK? Should v1.* platform work on JDK v1.6?